src/llmq/quorums_signing.cpp

```diff
@@ -440,11 +440,25 @@ void CSigningManager::CollectPendingRecoveredSigsToVerify(
     }
 }
 
+bool CSigningManager::ProcessPendingReconstructedRecoveredSigs()
```

Comment:
```suggestion
void CSigningManager::ProcessPendingReconstructedRecoveredSigs()
```
or maybe `ProcessRecoveredSig` should return `bool` and we should return early if it's `false` (and in `ProcessPendingRecoveredSigs` too)?